lib/crypto/test/crypto_SUITE.erl

```diff
@@ -1242,6 +1244,45 @@ cipher_info(Config) when is_list(Config) ->
             ct:fail('Cipher unsupported',[])
     end.
                                                                          
+cipher_info_aead_attr() ->
+    [{doc, "crypto cipher_info aead attribute testing"}].
+cipher_info_aead_attr(Config) when is_list(Config) ->
+    AeadCiphers = [aes_128_ccm, aes_192_ccm, aes_256_ccm, aes_128_gcm, aes_192_gcm, aes_256_gcm, chacha20_poly1305],
+    case lists:foldl(fun(C,Ok) ->
+                        case crypto:cipher_info(C) of
+                            #{aead := true} ->
+                                true;
```

Comment:
```suggestion
                                true and Ok;
```
so one 'false' make the accumulator stay 'false'?  Alternatively use lists:any/2 or lists:all/2.